code/modules/planar_lighting/light_effect.dm

```diff
@@ -13,18 +13,38 @@ var/list/light_over_cache = list()
 	glide_size = 32
 	blend_mode = BLEND_ADD
 
-	var/image/light_overlay
+	light_range = 5
+
+//	var/image/light_overlay
 	var/current_power = 1
 	var/atom/movable/holder
 	var/point_angle
 	var/list/affecting_turfs = list()
 
 /obj/light/New(var/newholder)
 	holder = newholder
-	light_overlay = image(icon = 'icons/planar_lighting/lighting_overlays.dmi', icon_state = holder.light_type)
+
+	light_range = max(0, min(5, light_range))
+	pixel_x = pixel_y = -(world.icon_size * light_range)
```

Comment:
Yea, I know, I'm going to add 511 to the Travis file in the next commit.

On 13 Sep 2016 23:52, GinjaNinja32 <notifications@github.com> wrote:

In code/modules/planar_lighting/light_effect.dm<https://github.com/Yonaguni/EuropaStation/pull/473#discussion_r78543145>:

>       var/current_power = 1
>       var/atom/movable/holder
>       var/point_angle
>       var/list/affecting_turfs = list()
>
>  /obj/light/New(var/newholder)
>       holder = newholder
> -     light_overlay = image(icon = 'icons/planar_lighting/lighting_overlays.dmi', icon_state = holder.light_type)
> +
> +     light_range = max(0, min(5, light_range))
> +     pixel_x = pixel_y = -(world.icon_size * light_range)


x = y = z is only supported in 511+; this will need to be x = z; y = x or similar to run on 510.

—
You are receiving this because you were assigned.
Reply to this email directly, view it on GitHub<https://github.com/Yonaguni/EuropaStation/pull/473/files/c45d254157879c864b0413b782fa18d18ebfa0dd#r78543145>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AGEAeFAhw_2a4fnzgjJrvbtMbJbsjS9lks5qpo5WgaJpZM4J6N-z>.